sources/patches/app-1.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/lib/tasks/assets.rake b/lib/tasks/assets.rake
```

Comment:
Maybe could you give a more explicit patch name (e.g. app-1-use-latest_uglify-js.patch)...? :blush:

Reply:
Yes, good idea,

Should be fixed